ykman/cli/fido.py

```diff
@@ -176,23 +219,54 @@ def try_reset(controller_type):
             controller = ctx.obj['controller']
             controller.reset(touch_callback=prompt_for_touch)
 
-    try:
-        try_reset(Fido2Controller)
-    except CtapError as e:
-        if e.code == CtapError.ERR.ACTION_TIMEOUT:
-            ctx.fail(
-                'Reset failed. You need to touch your'
-                ' YubiKey to confirm the reset.')
-        elif e.code == CtapError.ERR.NOT_ALLOWED:
-            ctx.fail(
-                'Reset failed. Reset must be triggered within 5 seconds'
-                ' after the YubiKey is inserted.')
-        else:
+    if ctx.obj['dev'].is_fips:
+        if not force:
+            click.echo(
```

Comment:
This `click.echo` could me merged with the `click.prompt` below, no?

Reply:
Sure.